client/components/dataviews/style.scss

```diff
@@ -195,3 +199,38 @@
 		display: none;
 	}
 }
+
+// TODO: remove when updating to dataviews@4.3.0.
+//
+// This is a hotfix to hiding the properties section in the view config menu
+// when no field is hidable.
+// This has been fixed upstream as of
+// https://github.com/WordPress/gutenberg/pull/64711/files
+.dataviews-settings-section:has(.dataviews-settings-section__content:empty) {
+	display: none;
+}
+
+// TODO: remove when updating to dataviews@4.3.0
+//
+// This is a hotfix for the sort direction control within the view config.
+// It shouldn't be visible when there are no sortable fields.
+// Fixed at https://github.com/WordPress/gutenberg/issues/64816
+.is-section-a8c-for-agencies-team .dataviews-settings-section__content .is-divided-in-two,
+.is-section-a8c-for-agencies-sites.modal-open .dataviews-settings-section__content .is-divided-in-two,
+.is-section-a8c-for-agencies-referrals .dataviews-settings-section__content .is-divided-in-two {
+	display: none !important;
+}
+
+// TODO: remove when updating dataviews to have this built-in.
+//
+// https://github.com/Automattic/wp-calypso/pull/93503#discussion_r1734341411
+.dataviews-filters-toggle__count {
+	padding: revert !important;
+}
+
+// TODO: remove when this is fixed upstream.
+//
+// https://github.com/WordPress/gutenberg/issues/64925
```

Comment:
Listing here the PRs/Issues related to the hotfixes for cross-reference:

- https://github.com/WordPress/gutenberg/issues/64925
- https://github.com/Automattic/wp-calypso/pull/93503#discussion_r1734341411
- https://github.com/WordPress/gutenberg/issues/64816
- https://github.com/WordPress/gutenberg/pull/64711